apps/api/serializers.py

```diff
@@ -1,6 +1,12 @@
+from django.contrib.auth import get_user_model
```

Comment:
User model is not used in this file, please remove it.

Reply:
yes you're right